tests/test_basic.py

```diff
@@ -47,9 +49,23 @@ def test_silhouette_batch():
     scib_metrics.silhouette_batch(X, labels, batch)
 
 
-def test_nmi_ari_cluster_labels():
+def test_nmi_ari_cluster_labels_kmeans():
     X, labels = dummy_x_labels()
-    nmi, ari = scib_metrics.nmi_ari_cluster_labels(X, labels)
+    nmi, ari = scib_metrics.nmi_ari_cluster_labels_kmeans(X, labels)
+    assert isinstance(nmi, float)
+    assert isinstance(ari, float)
+
+
+def test_nmi_ari_cluster_labels_leiden_parallel():
+    X, labels = dummy_x_labels(return_symmetric_positive=True)
+    nmi, ari = scib_metrics.nmi_ari_cluster_labels_leiden(X, labels, optimize_resolution=True, n_jobs=2)
```

Comment:
is it possible to check this against some leiden impl to make sure the clusters are the same? Is this reliable for diff random seeds?

Reply:
ah jk i see its leiden vs louvain

Reply:
leiden is faster. And the igraph implementation is even faster than the one scanpy uses